### PR TITLE
feat: add streaming automl classification samples

### PR DESCRIPTION
**Pull request by leahecole**

### CODE REVIEW

**Comment by leahecole** [comment]
Not sure what is invalid about the license header - it's the same as any other license header in the project, and when I follow the links to go to GCF I get a "Bad Request" - any insights related to that or the PR in general are appreciated!

**Comment by nnegrey** [comment]
Good License: `Copyright 2019 Google LLC`

No periods, or commas or anything else. Just the above only.
Bad License: 
- `Copyright 2019 Google, LLC.`
- `Copyright 2019 Google LLC.`
- `Copyright 2019, Google, LLC.`
- etc.

**Comment by leahecole** [comment]
The model is in the wrong project - it's in nodejs-docs-tests instead of google-cloud-node. I'll kick off the training process and make the change once it's done 🤦‍♀

File: samples/analyze-streaming-automl-classification.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const modelPath = `projects/${projectId}/locations/us-central1/models/${modelId}`;
+  const configRequest = {
+    videoConfig: {
+      feature: 'STREAMING_AUTOML_CLASSIFICATION',
+      automlClassificationConfig: {
+        modelName: modelPath,
+      },
+    },
+  };
+  const readStream = fs.createReadStream(path, {
```
Comment (anguillanneuf) [inline samples/analyze-streaming-automl-classification.js]: We need a comment about video format for the input file here. And link to the help page if people get stuck.

**Pull request merged**